package.json

```diff
@@ -54,11 +54,11 @@
     "http-string-parser": "^0.0.6",
     "husky": "^4.3.8",
     "jest": "^27.2.5",
-    "lerna": "^4.0.0",
+    "lerna": "5.1.4",
```

Comment:
this was the only change that scared me a little since it's a big jump. But I think we only use lerna for releasing, so not sure how to test this prior to trying to release a new version of Prism 😬

Reply:
looks like we wouldn't be affected: https://github.com/lerna/lerna/blob/main/CHANGELOG.md#breaking-changes

Reply:
nice! Hadn't gotten around to looking at the docs for this just yet. That's promising!